client/components/editor/teaching-elements/Assessment/TextSelection/index.vue

```diff
@@ -19,18 +19,13 @@
     </div>
     <div v-if="isEditing">
       <button
-        v-if="isSelecting"
-        @click="edit"
-        class="btn btn-primary btn-material">
-        <span class="mdi mdi-stop-circle-outline"/>
-          Edit content
-      </button>
-      <button
-        v-else
-        @click="select"
-        class="btn btn-primary btn-material">
-        <span class="mdi mdi-marker"/>
-          Select solutions
+        @click="isSelecting ? edit() : select()"
+        class="btn btn-primary btn-material"
+        type="button">
+        <span
+          :class="isSelecting ? 'mdi-stop-circle-outline' : 'mdi-marker'"
+          class="mdi"/>
```

Comment:
Those are **only** html elements that can be self closed:
https://html.spec.whatwg.org/multipage/syntax.html#void-elements
No `<span>` here ❕